lib/btrfs_test.pm

```diff
@@ -21,8 +21,14 @@ sub set_playground_disk {
             $vd = 'sd';
         }
         assert_script_run 'parted --script --machine -l';
-        my $disk = script_output 'parted --script --machine -l |& sed -n \'s@^\(/dev/' . $vd . '[ab]\):.*unknown.*$@\1@p\'';
```

Comment:
Did you see how we do it in tests/console/zypper_lifecycle.pm now? Should do the same but we could stick to the same way of writing it ->

```perl
my $output = script_output 'parted --script --machine -l';
$output =~ s|(/dev/$vd[ab]):.*unknown.*|$1|s || die "Failed to parse playground disk, got following output:\n$output";
```

Reply:
Changed